<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CvqMotion</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="DeviceControlWidget" name="deviceControlWidget" native="true"/>
        </item>
        <item>
         <widget class="cvqm::CameraView" name="cameraView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>150</verstretch>
           </sizepolicy>
          </property>
          <property name="baseSize">
           <size>
            <width>640</width>
            <height>480</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>150</height>
        </size>
       </property>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionView_Source"/>
    <addaction name="actionView_Blur"/>
    <addaction name="actionView_Background"/>
    <addaction name="actionView_Background_Delta"/>
    <addaction name="actionView_Foreground_Threshold"/>
    <addaction name="actionView_Dilated_Foreground"/>
    <addaction name="actionView_Dilated_Background"/>
    <addaction name="actionView_Output"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionMotion_Detection_Parameters"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuDebug"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Camera  Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionAdd_Detection_Zone"/>
   <addaction name="actionAdd_Mask_Out_Zone"/>
   <addaction name="actionRemove_Zone"/>
   <addaction name="actionMeasure_Distance"/>
  </widget>
  <action name="actionView_Source">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Source</string>
   </property>
  </action>
  <action name="actionView_Blur">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Blur</string>
   </property>
  </action>
  <action name="actionView_Background">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Background</string>
   </property>
  </action>
  <action name="actionView_Background_Delta">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Background Delta</string>
   </property>
  </action>
  <action name="actionView_Foreground_Threshold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Thresholded Foreground</string>
   </property>
  </action>
  <action name="actionView_Dilated_Foreground">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Dilated Foreground</string>
   </property>
  </action>
  <action name="actionView_Original_Frame">
   <property name="text">
    <string>View Original Frame</string>
   </property>
  </action>
  <action name="actionView_Dilated_Background">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Dilated Blending</string>
   </property>
  </action>
  <action name="actionView_Output">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Output</string>
   </property>
  </action>
  <action name="actionAdd_Detection_Zone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add Detection Zone</string>
   </property>
   <property name="toolTip">
    <string>Add a new detection zone by drawing it on the camera view</string>
   </property>
  </action>
  <action name="actionAdd_Mask_Out_Zone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add Mask-Out Zone</string>
   </property>
   <property name="toolTip">
    <string>Add a new ignore zone by drawing it on the camera view</string>
   </property>
  </action>
  <action name="actionRemove_Zone">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Remove Zone</string>
   </property>
   <property name="toolTip">
    <string>Remove a zone by clicking within it on the camera view</string>
   </property>
  </action>
  <action name="actionMeasure_Distance">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Measure Distance</string>
   </property>
   <property name="toolTip">
    <string>Measure the distance between two points on the camera view</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionMotion_Detection_Parameters">
   <property name="text">
    <string>Motion Detection Parameters</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionView_Thresholded_Background">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>View Thresholded Blending</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>cvqm::CameraView</class>
   <extends>QGraphicsView</extends>
   <header>cameraview.h</header>
  </customwidget>
  <customwidget>
   <class>DeviceControlWidget</class>
   <extends>QWidget</extends>
   <header>devicecontrolwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>349</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
